pages/auth.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login | Cickstock</title>
```

Comment:
Crickstock, Miss spelled I think

Reply:
Thanks for pointing it out, I have fixed it in the latest commit